example/coreapi/core_api_credit_card_example.rb

```diff
@@ -15,29 +15,34 @@
 
 # prepare CORE API parameter to get credit card token
 # another sample of card number can refer to https://docs.midtrans.com/en/technical-reference/sandbox-test?id=card-payments
-card = {
-  card_number: 5211111111111117,
-  card_cvv: 123,
-  card_exp_month: 12,
-  card_exp_year: 2025
-}
-get_token = Midtrans.create_card_token(card)
-
-# prepare CORE API parameter to charge credit card ( refer to: https://docs.midtrans.com/en/core-api/credit-card?id=_2-sending-transaction-data-to-charge-api )
-result = Midtrans.charge(
-  {
-    "payment_type": "credit_card",
-    "transaction_details": {
-      "gross_amount": 10000,
-      "order_id": "ruby-example-coreapi-creditcard-#{Time.now.to_i}"
-    },
-    "credit_card": {
-      "token_id": "#{get_token.token_id}"
-    }
+begin
+  card = {
+    card_number: 5211111111111117,
+    card_cvv: 123,
+    card_exp_month: 12,
+    card_exp_year: 2025
   }
-)
+  get_token = Midtrans.create_card_token(card)
 
-puts result.data
+  # prepare CORE API parameter to charge credit card ( refer to: https://docs.midtrans.com/en/core-api/credit-card?id=_2-sending-transaction-data-to-charge-api )
+  result = Midtrans.charge(
+    {
+      "payment_type": "credit_card",
+      "transaction_details": {
+        "gross_amount": 10000,
+        "order_id": "ruby-example-coreapi-creditcard-#{Time.now.to_i}"
+      },
+      "credit_card": {
+        "token_id": "#{get_token.token_id}"
+      }
+    }
+  )
+  puts "charge result : #{result.data}"
+rescue MidtransError => e
+  puts e.message # Basic message error
+  puts e.status # HTTP status code e.g: 400, 401, etc.
+  puts e.data # JSON of the API response
```

Comment:
It is not JSON in this case right? Shouldn't it be `API response body in String` ?